<launch>
  <node name="nvilidar_node"  pkg="nvilidar_ros"  type="nvilidar_node" output="screen" respawn="false" >
    <param name="serialport_name"         type="string" value="/dev/nvilidar"/>  
    <param name="serialport_baud"         type="int" value="921600"/>  
    <param name="ip_addr"                 type="string" value="192.168.1.200"/> 
    <param name="lidar_udp_port"          type="string" value="8100"/> 
    <param name="config_udp_port"         type="string" value="8200"/> 
    <param name="frame_id"                type="string" value="laser_frame"/>
    <param name="resolution_fixed"        type="bool"   value="true"/>
    <param name="auto_reconnect"          type="bool"   value="false"/>
    <param name="reversion"               type="bool"   value="false"/>
    <param name="inverted"                type="bool"   value="false"/>
    <param name="angle_min"               type="double" value="-180" />
    <param name="angle_max"               type="double" value="180" />
    <param name="range_min"               type="double" value="0.1" />
    <param name="range_max"               type="double" value="64.0" />
    <param name="aim_speed"               type="double" value="30.0"/>
    <param name="sampling_rate"           type="int" value="30"/>
    <param name="sensitive"               type="bool" value="false"/>
    <param name="tailing_level"           type="int" value="6"/>
    <param name="single_channel"          type="bool" value="false"/>
    <param name="ignore_array_string"     type="string" value="" />
  </node>
  <node pkg="tf" type="static_transform_publisher" name="base_link_to_laser4"
    args="0.19 0.0 0.2 0.06 0.0  0.0 /base_footprint /laser_frame 40" />
</launch>
   
